spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/influx/InfluxDbAutoConfigurationTests.java

```diff
@@ -61,6 +65,22 @@ public void influxDbCanBeCreatedWithoutCredentials() {
 		assertThat(this.context.getBeansOfType(InfluxDB.class)).hasSize(1);
 	}
 
+	@Test
+	public void influxDbWithOkHttpClientBuilder() {
+		load(Config.class, "spring.influx.url=http://localhost",
+				"spring.influx.password:password", "spring.influx.user:user");
+		assertThat(this.context.getBeansOfType(InfluxDB.class)).hasSize(1);
+		assertThat(this.context.getBeansOfType(OkHttpClient.Builder.class)).hasSize(1);
```

Comment:
That doesn't test anything is it? Can't we check that the `InfluxDB` instance has been customized with the state of the builder instead? I know this is similar to other tests in this case (a shame :( ) but perhaps this is an opportunity for some improvements?